src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -247,6 +247,22 @@ namespace Sapphire::Network::Packets::Server
     } entry[8];
   };
 
+  struct FFXIVIpcFreeCompanyMemberLogInOut : FFXIVIpcBasePacket< FreeCompanyMemberLogInOut >
+  {
+    uint16_t unknown;
+    uint16_t unknown1;
+    uint16_t unknown2;
+    uint16_t unknown3;
+    uint16_t unknown4;
+    char padding[6];
+    uint8_t status; // 0x0F Login, 0x10 Logout
```

Comment:
do the other status codes do other things? this could be the same packet for the messages like executing company actions, people joining/leaving the fc, etc meaning that `FreeCompanyMemberLogInOut` isn't exactly correct